tests/tests/regional_atmaq

```diff
@@ -100,6 +100,6 @@ export CCPP_SUITE=FV3_GFS_v15p2
 
 if [[ $MACHINE_ID = hera ]]; then
   TPN=30
-elif [[ $MACHINE_ID = cheyenne || $MACHINE_ID = gaea ]]; then
+elif [[ $MACHINE_ID = derecho || $MACHINE_ID = gaea ]]; then
```

Comment:
What is TPN on Derecho? Is it similar to gaea? Using TPN=16 when more would still make the test pass is a waste of resources. These three tests should not use reduced TPN unless it is shown to be required.

Reply:
I mis-read the documentation for Derecho. It says

``Dual-socket nodes, 64 cores per socket``

So TPN=128, which is what default_vars is correctly using. Do we really want to set some tests to TPN=16?

Reply:
I found documentation that states the total TPN on Derecho is 128. In any case, it's probably worth it to remove these for Derecho on the test files and when I retest we can confirm if they are needed or not.

Reply:
Removed rest of special TPNS for Derecho. If we find necessary, we can add later.